src/backends/mod.ts

```
@@ -27,6 +28,18 @@ export async function resolveHandler(
   }
 
+  try {
+    await verifyV4Signature(
+      c.req.raw,
+      globalConfig.buckets[bucketName].config,
+      {
+        // FIXME: properly source the credential lists
+      },
+    );
+  } catch (err) {
+    throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)